src/samVisualize/activation.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { SamVisualizationManager } from './commands/visualizeSamTemplate/samVisualizationManager'
+
+export async function activate(extensionContext: vscode.ExtensionContext): Promise<void> {
+    registerTemplateVisualCommands(extensionContext)
+}
+
+async function registerTemplateVisualCommands(extensionContext: vscode.ExtensionContext): Promise<void> {
+    const visualizationManger = new SamVisualizationManager(extensionContext)
+    extensionContext.subscriptions.push(
+        vscode.commands.registerCommand('aws.samVisualize.renderTemplate', async () => {
+            return visualizationManger.renderSamVisualization(vscode.window.activeTextEditor)
+        })
+    )
+}
```

Comment:
why we need two functions if the 1st's sole purpose is to call the 2nd one?

Reply:
We don't, really. At the time of writing I thought more would have to be done in activation, but we only need one command.